Wrappers/Python/ccpi/viewer/utils/colormaps.py

```diff
@@ -1114,27 +1114,31 @@ def relu(x, xmin, xmax, scaling=1):
 
 class CILColorMaps(object):
     @staticmethod
-    def get_color_transfer_function(cmap, color_range):
-
-        tf = vtk.vtkColorTransferFunction()
-
-
+    def get_color_map(cmap):
```

Comment:
Are changes to this file part of the webapp? Or these changes are already present in master?

Reply:
This is part of the viewer in the sense that it is a small refactoring that allows calling for the color_map without a color_range